publish/lib/telemetry.js

```diff
@@ -1,9 +1,16 @@
 import { InfluxDB, Point } from '@influxdata/influxdb-client'
 
+const {
+  INFLUXDB_TOKEN
+} = process.env
+
+if (!INFLUXDB_TOKEN) {
+  console.warn('Warning: INFLUXDB_TOKEN was not provided by the environment')
+}
+
 const influx = new InfluxDB({
   url: 'https://eu-central-1-1.aws.cloud2.influxdata.com',
-  // spark-publish-write
-  token: 'Zkqa_s7mI0W_WKI6DUmu-iRnQkCvwNaQfbPK_zT7I6iYYaC2C1kokdlhO2jb4tjRcAJQHQXAGnrdD3vqlMZ63g=='
```

Comment:
@juliangruber could you please invalidate (delete) this token in InfluxDB? I could not find it in my account.

Reply:
deleted 👍